src/loaders/gtdb/parse_computed_genome_attribs.py

```diff
@@ -0,0 +1,223 @@
+"""
+PROTOTYPE
+
+This script involves processing computed genome attributes and organizing them into a structured format suitable
+for importing into ArangoDB. The resulting JSON file will be used to upsert the database with the parsed genome
+attribute data.
+
+Note: The database was previously created using a JSON file generated by the gtdb_genome_attribs_loader.py script.
+      It is important to ensure that the arguments "--load_ver" and "--kbase_collection" match the ones used in the
+      gtdb_genome_attribs_loader.py script in order to generate the same key for the corresponding Arango document.
+
+usage: parse_computed_genome_attribs.py [-h] --tools TOOLS [TOOLS ...] --load_ver LOAD_VER
+                                        [--kbase_collection KBASE_COLLECTION] [--root_dir ROOT_DIR] [-o OUTPUT]
+options:
+  -h, --help            show this help message and exit
+
+required named arguments:
+  --tools TOOLS [TOOLS ...]
+                        Extract results from tools. (e.g. gtdb_tk, checkm2, etc.)
+  --load_ver LOAD_VER   KBase load version (e.g. r207.kbase.1). This argument should be consistent with the one used by the
+                        gtdb_genome_attribs_loader.py script.
+
+optional arguments:
+  --kbase_collection KBASE_COLLECTION
+                        KBase collection identifier name (default: loader_common_names.DEFAULT_KBASE_COLL_NAME).
+                        This argument should be consistent with the one used by the gtdb_genome_attribs_loader.py script.
+  --root_dir ROOT_DIR   Root directory.
+  -o OUTPUT, --output OUTPUT
+                        output JSON file path (default: gtdb_computed_genome_attribs.json)
+
+"""
+import argparse
+import copy
+import os
+import sys
+
+import pandas as pd
+
+from src.loaders.common import loader_common_names
+from src.loaders.gtdb.gtdb_loader_helper import convert_to_json, init_genome_atrri_doc
+
+# Default result file name for parsed computed GTDB genome attributes data for arango import
+GTDB_COMPUTED_GENOME_ATTR_FILE = "gtdb_computed_genome_attribs.json"
```

Comment:
Why is this gtdb exclusive? Shouldn't this script work for PMI etc?

Reply:
updated.

Reply:
If this is a script that can be run for any collection maybe it shouldn't go in the `gtdb` folder?

Reply:
...and maybe some of the helper routines should be moved to `loaders/common`

Reply:
Subsumed in subsequent review